Formula/glib.rb

```diff
@@ -62,6 +63,21 @@ def install
               "giomoduledir=#{HOMEBREW_PREFIX}/lib/gio/modules",
               "giomoduledir=${libdir}/gio/modules"
 
+    # Delete python files because they are provided by `glib-utils`
+    dummy_python_shebang_rewrite_info = python_shebang_rewrite_info("#{HOMEBREW_PREFIX}/bin/python3")
```

Comment:
```suggestion
    dummy_python_shebang_rewrite_info = python_shebang_rewrite_info("/usr/bin/env python3")
```
We no longer guarantee that the user has a Homebrew Python installed and linked.

Reply:
We do not use the exact value of the Python executable path. We only need the `rewrite_info.regex` here. That's why the variable is named "dummy".

Reply:
Just create a `Regexp` object for that purpose then. There's no need for the code to be this opaque.

Reply:
Do you mean copy-paste the regex from `Language::Python::python_shebang_rewrite_info` to here?

```ruby
      # @private
      def python_shebang_rewrite_info(python_path)
        Utils::Shebang::RewriteInfo.new(
          %r{^#! ?/usr/bin/(?:env )?python(?:[23](?:\.\d{1,2})?)?( |$)},
          28, # the length of "#! /usr/bin/env pythonx.yyy "
          "#{python_path}\\1",
        )
      end
```

what if it has been updated in the future in Homebrew/brew.

Reply:
> Do you mean copy-paste the regex from `Language::Python::python_shebang_rewrite_info` to here?

Yes.

> what if it has been updated in the future in Homebrew/brew.

What if it is updated to not have `#regex` attribute at all? The [documentation](https://rubydoc.brew.sh/Utils/Shebang/RewriteInfo.html) clearly indicates that this is a private API, and should not be used outside of Homebrew/brew.